MapboxDirectionsTests/V5Tests.swift

```diff
@@ -69,6 +69,8 @@ class V5Tests: XCTestCase {
         let leg = route!.legs.first!
         XCTAssertEqual(leg.name, "I 80, I 80;US 30")
         XCTAssertEqual(leg.steps.count, 59)
+        let roadClasses = leg.steps.first!.intersections!.first!.roadClasses!
+        XCTAssertTrue(roadClasses.contains([.toll, .restricted]))
```

Comment:
To prevent the test harness from crashing when any of these things is nil, use optionals:

```swift
let firstStep = leg.steps.first // as opposed to step below
XCTAssertNotNil(firstStep)
let intersections = firstStep?.intersections
XCTAssertNotNil(intersections)
let intersection = intersections?.first
XCTAssertNotNil(intersection)
let roadClasses = intersection?.roadClasses
XCTAssertNotNil(roadClasses)
XCTAssertTrue(roadClasses?.contains([.toll, .restricted]) ?? false)
```